components/ClubPage/InfoBox.js

```diff
@@ -39,7 +39,7 @@ const InfoBox = props => {
   ]
 
   return data.map(({ icon, text }) => (
-    <p style={infoStyles}>
+    <p style={infoStyles} key={text}>
```

Comment:
Can we use the shared `Text` component from the shared components?

Reply:
I don't think this was introduced by me, but instead upstream in #59. Can fix in this merge request, just note that the same issue will appear in #59.